bin/jest.js

```diff
@@ -113,6 +117,22 @@ var argv = optimist
       ),
       type: 'boolean'
     },
+    watch: {
+      alias: 'm',
```

Comment:
probably remove it since `w` is used for `maxWorkers` and I was thinking `m`(stands for `monitor) can be a good alternative